sdk/eventhub/Azure.Messaging.EventHubs/samples/README.md

```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```

Comment:
Was this a BOM issue?    I'm not seeing content changed.

Reply:
Sorry for the delayed response. It was just a removal of a space or a newline; I don't remember which. There was no content change.